proto/babylon/finality/v1/params.proto

```diff
@@ -30,4 +30,7 @@ message Params {
   // jail_duration is the minimum period of time that a finality provider remains jailed
   google.protobuf.Duration jail_duration = 5
   [(gogoproto.nullable) = false, (amino.dont_omitempty) = true, (gogoproto.stdduration) = true];
+  // activation_block_height is the babylon block which the finality module will start to
+  // accept public randomness commit and finality voting.
+  uint64 activation_block_height = 6;
```

Comment:
Hmm wouldn't this be better suited as a constant rather than a parameter? Since

- This value will only be used until a given height
- It might be possible that a gov prop to update this value to a very big one, to disable finality. This sounds like an attack vector to me.

Reply:
There was a conversation during the friday meeting that this value should be configurable that might change depending on the chain finality registration and amount of btc transitioned

Reply:
@KonradStaniec yes, we could add to the upgrade handler, mainnet I will set to 48hrs after upgrade ~17500 blocks and for testnet set to a low number to not hold the e2e test too much, also will add a check for it in the e2e testing

Thanks for the remainder

Reply:
on last cross-team meeting (btw. do you have invite to this one ? it is the same time as core meeing but on Friday) we decided to make this a parameter as it may happen that we won't gather enough stake before the activation height we hard code, which would be critical scenario. When it is parameter, it would be just a question of gov proposal to postpone finality activation a bit.

> It might be possible that a gov prop to update this value to a very big one, to disable finality. This sounds like an attack vector to me.

wouldn't that require >66% of the native stake being malicious ? i.e if that happens we have other problems 😅 and at the beginning of the Babylon chain, it will be Babylon foundation who controls most stake.